contributing-guides/style-guide.md

```diff
@@ -34,10 +34,10 @@ tldrl -f {{page.md}}
 For other ways to use `tldrl`, such as linting an entire directory, check out (what else!)
 [`tldr tldrl`](https://github.com/tldr-pages/tldr/blob/main/pages/common/tldrl.md)
 
-If you're using the Node.js client of tldr-pages, you can preview a page locally using the `-f` flag (aka `--render`):
+If you're using the Node.js or Python clients of tldr-pages, you can preview a page locally using the `--render` argument (aka `-f` in Node.js, and `-r` in Python):
```

Comment:
Is there a reason you replaced `flag` with `argument`?

Reply:
I think `flag` is used for one letter arguments, but the correct term is `command line argument`.

Reply:
Oh, interesting